### ISSUE TITLE
on refresh its hitting backend api for data

### ISSUE THREAD

**Issue by ErBahuguna** [AlbinoDrought/cachios#24]
I am using cachios in vuejs

import axios from 'axios';
import cachios from 'cachios';

cachios.getResponseCopy = function (response) {
  console.log(['cachiosInstance',response]);
  return {
    status: response.status,
    statusText: response.statusText,
    data: response.data,
  };
};

cachios.getCacheIdentifier = function (config) {
  console.log(['getCacheIdentifier',config]);
  return {
    method: config.method,
    url: config.url,
    params: config.params,
    data: config.data,
    headers: config.headers,
  };
};

this.requestAuthHeader={ headers: 
        { 
          'Content-Type': 'application/json',
          'Authorization': `${access_token}`
        },
        ttl: 180, // persist this result for 3 minutes 
      };

cachios.get(url,this.requestAuthHeader);

but when i refresh the page it still hit the backend api as i am setting 3 minutes for cache expire time.

**Comment by ggedde** [comment]
Yeah, same for me.  I see the cached item when inspecting the cachios object, but it is still hitting the URL to get the json data.  My expectations would be that it wouldn't.

I just did a fresh install and using Nuxt and Cachios and still get the issue.
